### PR TITLE
topaz: Initial topaz support for crDroid 11.x

### PR DESCRIPTION
**Pull request by mirabelle1999**

### CODE REVIEW

File: topaz.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    "oem": "Redmi",
+                    "device": "Note 12 4G/NFC",
+                    "filename": "crDroidAndroid-15.0-20241107-topaz-v11.0-BETA.zip",
+                    "download": "https://sourceforge.net/projects/crdroid/files/topaz/11.x/crDroidAndroid-15.0-20241107-topaz-v11.0-BETA.zip/download",
+                    "timestamp": 1730969752,
+                    "md5": "935c0ef576408a252f266065699d25a8",
+                    "sha256": "8f6dd3922dbdffcad1e44ed2d2c4c8bab4ea0ff409f699a6eb45daf875b5469f",
+                    "size": 1569729108,
+                    "version": "11.0",
+                    "buildtype": "Monthly",
```
Comment (gwolf2u) [inline topaz.json]: BETA

File: topaz.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    "forum": "",
```
Comment (gwolf2u) [inline topaz.json]: Missing

File: topaz.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    "gapps": "https://nikgapps.com/downloads",
+                    "firmware": "https://xmfirmwareupdater.com/archive/firmware/topaz/",
+                    "modem": "",
+                    "bootloader": "",
+                    "recovery": "https://orangefox.download/device/topaz",
+                    "paypal": "",
+                    "telegram": "https://t.me/mirabelle1999",
+                    "dt": "https://github.com/crdroidandroid/android_device_xiaomi_topaz",
+                    "common-dt": "",
+                    "kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_topaz"
```
Comment (gwolf2u) [inline topaz.json:25]: missing 15.0

File: topaz_changelog.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Highlights & Device Specific Changes:
+Build type: Monthly
```
Comment (gwolf2u) [inline topaz_changelog.txt]: BETA

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: topaz.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+                    "device": "Note 12 4G/NFC",
+                    "filename": "crDroidAndroid-15.0-20241107-topaz-v11.0-BETA.zip",
+                    "download": "https://sourceforge.net/projects/crdroid/files/topaz/11.x/crDroidAndroid-15.0-20241107-topaz-v11.0-BETA.zip/download",
+                    "timestamp": 1730969752,
+                    "md5": "935c0ef576408a252f266065699d25a8",
+                    "sha256": "8f6dd3922dbdffcad1e44ed2d2c4c8bab4ea0ff409f699a6eb45daf875b5469f",
+                    "size": 1569729108,
+                    "version": "11.0",
+                    "buildtype": "BETA",
+                    "forum": "https://xdaforums.com/t/rom-15-official-vanilla-crdroid-for-redmi-note-12-4g.4681116/",
```
Comment (gwolf2u) [inline topaz.json]: update forum template

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Comment by mirabelle1999** [comment]
XDA Link and trees are now updated sir

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**